cluster/replication/copier/copier.go

```diff
@@ -250,19 +250,37 @@ func (c *Copier) AsyncReplicationStatus(ctx context.Context, srcNodeId, targetNo
 		return models.AsyncReplicationStatus{}, err
 	}
 
+	if len(status.Shards) == 0 {
```

Comment:
thanks for improving the error message here, it will be really helpful for debugging